850.split-ambiguities/e.yaml

```diff
@@ -97,6 +97,12 @@
 
 - { name: enigma, wwwpart: sgtatham, setname: enigma-sgtatham }
 
+- { name: envision, wwwpart: gabmus, setname: envision-xr }
+- { name: envision, wwwpart: atari.miribilist.com, setname: envision-atari }
+- { name: envision, wwwpart: enlightenment.org, setname: envision-video-player }
+- { name: envision, ruleset: aur, setname: envision-ide } # AUR package doesn't have a homepage
```

Comment:
ruleset matchers are not allowed here. You can use wwpart: `dimitar-asenov` or `ethz.ch` as there are related homepages https://github.com/dimitar-asenov/Envision.